doc/classes/NavigationServer2D.xml

```diff
@@ -303,6 +303,13 @@
 				Returns all created navigation map [RID]s on the NavigationServer. This returns both 2D and 3D created navigation maps as there is technically no distinction between them.
 			</description>
 		</method>
+		<method name="is_baking" qualifiers="const">
```

Comment:
comment from @smix8 via chat: `for the server api make the function name more explicit to what it does, e.g. is_navigation_mesh_baking()`